records_mover/db/postgres/copy_options/date_input_style.py

```diff
@@ -50,6 +51,13 @@ def upgrade_date_order_style(style: DateOrderStyle, hint_name: str) -> None:
 
     # Default value is "YYYY-MM-DD HH:MI:SSOF".
 
+    # TODO: Does this need to be generalized to be more helpful to people?
+    #
```

Comment:
Here's a good example of code which needs to change in a following PR before we can generalize the spec - this code is about checking against supported values rather than converting from one format encoding language to another.

I don't believe it'll be a difficult conversion.